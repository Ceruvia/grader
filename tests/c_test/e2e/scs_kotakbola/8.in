480 341 826
B 31
H 81
B 81
M 98
B 73
M 45
H 69
H 43
B 40
M 94
H 92
B 17
B 15
B 43
H 19
M 16
E
