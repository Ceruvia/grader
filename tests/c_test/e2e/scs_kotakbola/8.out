304 253 300
Ya
